README.md

```diff
@@ -21,9 +24,68 @@ npm i
 ## Usage
 
 ```
-CLIENT_SECRET_DB=zMultibaseMultiencoded npm test
+npm test
 ```
 
 ## Implementation
-To add your implementation to this test suite see the [README here.](https://github.com/w3c-ccg/vc-api-test-suite-implementations)
-Add the tag `Ed25519Signature2020` to the issuers and verifiers you want tested.
+
+You will need an issuer and verifier that are compatible with [VC API](https://w3c-ccg.github.io/vc-api/)
+and are capable of handling issuance and verification of Verifiable Credentials
+with `Ed25519Signature2020` proof type.
+
+To add your implementation to this test suite, you will need to add 2 endpoints
+to your implementation manifest.
+- A credentials issuer endpoint (`/credentials/issue`) in the `issuers`
+  property.
+- A credentials verifier endpoint (`/credentials/verify`) in the `verifiers`
+  property.
+
+All endpoints will need the tag `Ed25519Signature2020`.
+
+A simplified manifest would look like this:
+
+```js
+{
+  "name": "My Company",
+  "implementation": "My implementation",
+  "issuers": [{
+    "id": "",
+    "endpoint": "https://issuer.mycompany.com/credentials/issue",
+    "method": "POST",
+    "tags": ["Ed25519Signature2020"]
+  }],
+  "verifiers": [{
+    "id": "",
+    "endpoint": "https://verifier.mycompany.com/credentials/verify",
+    "method": "POST",
+    "tags": ["Ed25519Signature2020"]
+  }]
+}
+```
+
+The example above represents an unauthenticated endpoint. You may add zcap or
+oauth authentication to your endpoints. You can find an example in the
+[vc-api-test-suite-implementations README here](https://github.com/w3c-ccg/vc-api-test-suite-implementations#adding-a-new-implementation).
+
+To run the tests, some implementations may require client secrets that can be
+passed as env variables to the test script. To see which ones require client
+secrets, please check the implementation manifest within the
+[vc-api-test-suite-implementations](https://github.com/w3c-ccg/vc-api-test-suite-implementations/tree/main/implementations) library.
+
+## Docker Integration (TODO)
+
+We are presently working on implementing a new feature that will enable the
+utilization of Docker images instead of live endpoints. The docker image that
+you provide will be started when the test suite is run. The image is expected
+to expose the API provided above, which will be utilized in the same way that
```

Comment:
```suggestion
use of Docker images instead of live endpoints. The Docker image that
you provide will be started when the test suite is run. The image is expected
to expose the API provided above, which will be used in the same way that
```